Mathlib/Analysis/Normed/Field/Basic.lean

```diff
@@ -265,6 +266,25 @@ theorem mulRight_bound (x : α) : ∀ y : α, ‖AddMonoidHom.mulRight x y‖ 
   exact norm_mul_le y x
 #align mul_right_bound mulRight_bound
 
+/-- A non-unital subalgebra of a non-unital seminormed ring is also a non-unital seminormed ring,
+with the restriction of the norm.
+
+See note [implicit instance arguments]. -/
```

Comment:
I can't find the note, and I don't see how it is relevant since all instance arguments below are usual instance arguments. Can you expand a little bit?

Reply:
Sorry, this was a copy-paste from the other docstrings in the file. I'll try and see if it still applies (seems likely not) and then I'll update all docstrings.